hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -1124,17 +1124,15 @@ public TenantUserList listUsersInTenant(String tenantName, String prefix)
   }
 
   @Override
-  public OmVolumeArgs getS3Volume() throws IOException {
+  public GetS3VolumeResponse getS3VolumeInfo() throws IOException {
     final GetS3VolumeRequest request = GetS3VolumeRequest.newBuilder()
         .build();
     final OMRequest omRequest = createOMRequest(Type.GetS3Volume)
         .setGetS3VolumeRequest(request)
         .build();
     final OMResponse omResponse = submitRequest(omRequest);
-    final GetS3VolumeResponse resp = handleError(omResponse)
-        .getGetS3VolumeResponse();
 
-    return OmVolumeArgs.getFromProtobuf(resp.getVolumeInfo());
+    return handleError(omResponse).getGetS3VolumeResponse();
```

Comment:
This class is called the `Translator` because it translates proto java objects into the desired workable java object. We shouldn't return proto generated objects here, but translate them into a class in the `org.apache.hadoop.ozone.om.helpers` package like the rest of the methods here.

Reply:
You are right. Changed return value to `S3VolumeInfo` (new helper class).